deploy.sh

```
@@ -1,5 +1,4 @@
 #!/bin/bash
-set -e
```

Why remove this, it means that errors will be silently ignored.

---

Several of the deploy scripts stopped halfway through last major one, I think this was the culprit so I removed them. You can add it back in I spose.

---

The culprit would be an error from one of the commands. You can ignore errors with `command || true` if needed.